src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -54,6 +57,7 @@
     private static String FLIGHT_TABLE = "flight";
     private static String FLIGHT_LOG_TABLE = "flightlog";
 
+    private static Logger logger = LoggerFactory.getLogger("bio.terra.stairway");
```

Comment:
```suggestion
    private static Logger logger = LoggerFactory.getLogger(FlightDao.class);
```

Can you switch this to the class version instead of the String? It is inconsistent in the code now but we are trying to stick to this pattern.

Reply:
Yes, I will change it and update the PR.